src/YieldDisburser.sol

```diff
@@ -55,17 +56,24 @@ contract YieldDisburser is OwnableUpgradeable {
         breadToken.claimYield(breadToken.yieldAccrued(), address(this));
 
         (uint256[] memory projectDistributions, uint256 totalVotes) = _getVotedDistribution(breadchainProjects.length);
-
         lastClaimedTimestamp = Time.timestamp();
         lastClaimedBlocknumber = Time.blockNumber();
 
         uint256 halfBalance = breadToken.balanceOf(address(this)) / 2;
         uint256 baseSplit = halfBalance / breadchainProjects.length;
+        uint256 percentage_of_total_vote;
```

Comment:
we should stick to either camelCase or snake_case. Most of the contract is in camelCase

Reply:
addressed with a50968d